src/screens/ProfileScreen.js

```diff
@@ -61,7 +61,7 @@ export default function ProfileScreen() {
         }
       />
 
-      <ActivityList isEmpty={isEmpty} isLoading={isLoading} network={network} sections={sections} {...accountTransactions} />
+      <ActivityList network={network} sections={sections} {...accountTransactions} />
```

Comment:
removed the isEmpty and isLoading props since the SectionList handles it internally.